scripts/README_STABLE_BUILD.md

```diff
@@ -1,10 +1,21 @@
 <h2 align="center">Prisma VS Code Extension</h2>
 <div align="center">
-  Adds syntax highlighting, linting, code completion, formatting, jump-to-definition and more for <a href="https://www.prisma.io/docs/concepts/components/prisma-schema">Prisma Schema</a> files.
+  Adds a database management UI, syntax highlighting, linting, code completion, formatting, jump-to-definition and more for <a href="https://www.prisma.io/docs/concepts/components/prisma-schema">Prisma Schema</a> files.
```

Comment:
The mention of the schema files feels a bit out of place now (and outdated considering the recent changes). Adjust that part also?